### PR TITLE
[Icons]: Update icons from Figma

### PR DESCRIPTION
**Pull request by brave-support-admin**
> @brave/leo@0.0.1 update-icons
> node ./src/scripts/update-icons.js

Ignoring icon rainbow-color.svg because:
- has a raster image element
Ignoring icon bitcoin.com-color.svg because:
- has a raster image element
Ignoring icon daum-color.svg because:
- has a raster image element

### CODE REVIEW

**Comment by aguscruiz** [comment]
Yeah, just for different icons in case we ever want to change the design of
one without affecting the other

**Review by AlanBreck — APPROVED** [review]

File: icons/price-tag.svg
Comment (AlanBreck) [inline icons/price-tag.svg:1]: @aguscruiz, is the background intentional?

Comment (aguscruiz) [reply icons/price-tag.svg:1]: Ugh, no, not intentional, must've clicked something by mistake. Fixed now

Comment (AlanBreck) [reply icons/price-tag.svg:1]: @aguscruiz, still didn't resolve it.

Comment (aguscruiz) [reply icons/price-tag.svg:1]: Hmmm, try one last time, I unchecked an export thing but there's nothing there that would show a bg

image

**Review by AlanBreck — APPROVED** [review]

File: icons/price-tag.svg
Comment (AlanBreck) [reply icons/price-tag.svg:1]: That icon is no longer included in the PR.

File: icons/country-au.svg
Comment (AlanBreck) [inline icons/country-au.svg:1]: @aguscruiz, something looks fishy here.

File: icons/price-tag.svg
Comment (AlanBreck) [reply icons/price-tag.svg:1]: @aguscruiz ☝️

Comment (aguscruiz) [reply icons/price-tag.svg:1]: Ok no problem, it's not used anywhere

File: icons/country-au.svg
Comment (aguscruiz) [reply icons/country-au.svg:1]: Fixed, there was another broken flag and fixed that one too

**Review by AlanBreck — CHANGES_REQUESTED** [review]
@aguscruiz, now the flags are gone.

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**